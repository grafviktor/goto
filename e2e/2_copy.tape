Set Shell bash
Enter
Type "gg -f temp"
Enter
Sleep 100ms
Type "n"
Type "network host"
Down
Ctrl+u
Type "127.0.0.1"
Down
Type "network host description"
Ctrl+S
Type "c"
Type "q"
