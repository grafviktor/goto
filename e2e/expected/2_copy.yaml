- host:
    title: network host
    description: network host description
    address: 127.0.0.1
- host:
    title: network host (1)
    description: network host description
    address: 127.0.0.1
